### PR TITLE
mention OLM in readme

### PR DESCRIPTION
**Pull request by vn971**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,6 +20,7 @@ The following are not yet supported:
  * Account registration
  * Room topics
  * Voice/video calling
+ * End-To-End encryption (OLM)
```
Comment (richvdh) [inline README.md]: s/OLM/Olm/ please :)

Comment (vn971) [inline README.md]: Sure. Probably the github issue (for Olm) can be linked here, too.

**Comment by vn971** [comment]
@richvdh done

**Comment by richvdh** [comment]
Thanks!

**Pull request merged**